lib/modules/add-scoped-id.js

```diff
@@ -43,14 +58,13 @@ const addScopedIdPlugin = postcss.plugin('add-scoped-id', options => {
   const selectorTransformer = selectorParser(selectors => {
     selectors.each(selector => {
       const target = getTargetNode(selector)
-
+      replaceDeepCombinator(selector)
```

Comment:
Why did you move this here?

Reply:
If it doesn't change the order, `/deep/` will be treated as a tag and be added with scoped id.